### PR TITLE
check for custom_classes

### PR DESCRIPTION
**Pull request by jmancewicz**
There's an error reading classification models from before custom_classes was added.

### CODE REVIEW

**Comment by jmancewicz** [comment]
@gheinrich, do you have any problem with this check?

File: digits/extensions/data/objectDetection/data.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def __init__(self, **kwargs):
         # this instance is automatically populated with form field
         # attributes by superclass constructor
 
-        if self.custom_classes != '':
+        if 'custom_classes' in self.__dict__ and self.custom_classes != '':
```
Comment (lukeyeager) [inline digits/extensions/data/objectDetection/data.py]: I think you want this instead:
```py
hasattr(self, 'custom_classes')
```

**Comment by gheinrich** [comment]
> @gheinrich, do you have any problem with this check?

None at all

File: digits/extensions/data/objectDetection/data.py (hunk 1 above)
Comment (jmancewicz) [inline digits/extensions/data/objectDetection/data.py]: I merged before I saw this. Making new PR.

**Pull request merged**